tests/unit/test_metrics.py

```diff
@@ -0,0 +1,50 @@
+from unittest.mock import Mock, patch
+
+import pytest
+from prometheus_client.samples import Sample
+
+import prometheus_juju_backup_all_exporter
+from prometheus_juju_backup_all_exporter.metrics import MetricBase
+
+
+class TestMetrics:
+    """Metrics test class."""
+
+    def test_cannot_init_metric_base(self):
+        """Test cannot initialize MetricBase."""
+        with pytest.raises(TypeError):
+            MetricBase()
+
+    @patch.object(
+        prometheus_juju_backup_all_exporter.metrics.MetricBase,
```

Comment:
Similarly here, you can omit the whole `prometheus_juju_backup_all_exporter.metrics` since `MetricBase` is directly imported